lib/SearchAndSort/SearchAndSort.js

```diff
@@ -235,6 +235,7 @@ class SearchAndSort extends React.Component {
     }).isRequired,
     syncQueryWithUrl: PropTypes.bool,
     title: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+    validateSearchOnSubmit: PropTypes.func, // should return bool. It prevents search submission on false
```

Comment:
`validate...` is one of those tricky function names that doesn't clearly indicate how a function behaves: does it return true/false, or return valid input/null, or return void/throw? 

The comment here perfectly clarifies this (thank you!) so you don't need to change anything. Just be careful with this construction, and think about if there is a way you could name things so that you don't have to explain them with a comment.